docs/platforms/javascript/common/migration/v7-to-v8/v8-new-performance-api.mdx

```diff
@@ -0,0 +1,253 @@
+
+These APIs are also available in `7.x` so you can incrementally moveover to the new APIs before upgrading to `8.x`.
+
+Previously, there were two key APIs for adding manual performance instrumentation to your applications:
+
+- `startTransaction()`
+- `span.startChild()`
+
+This showed the underlying data model that Sentry was originally based on, which is that there is a root Transaction which can have a nested tree of Spans.
+
+In the new model, transactions are conceptually gone. Instead, you will always operate on spans, no matter where in the tree you are. Note that in the background, spans may still be grouped into a transaction for the Sentry UI. However, this happens transparently, and from an SDK perspective, all you have to think about are spans.
+
+Instead of manually starting & ending transactions and spans, the new model does not differentiate between these two. Instead, you always use the same APIs to start a new span, and it will automatically either create a new Root Span (which is just a regular span, only that it has no parent, and is thus conceptually roughly similar to a transaction) or a Child Span for whatever is the currently active span.
+
+There are three key APIs available to start spans:
+
+- `startSpan()`
+- `startSpanManual()`
+- `startInactiveSpan()`
+
+All three span APIs take `StartSpanOptions` as a first argument, which has the following shape:
+
+```ts
+interface StartSpanOptions {
+  // The only required field - the name of the span
+  name: string;
+  attributes?: SpanAttributes;
+  op?: string;
+  scope?: Scope;
+  forceTransaction?: boolean;
+}
+```
+
+### `startSpan()`
+
+This is the most common API that should be used in most circumstances. It will start a new span, make it the active span
+for the duration of a given callback, and automatically end it when the callback ends. You can use it like this:
+
+```js
+Sentry.startSpan(
+  {
+    name: "my-span",
+    attributes: {
+      attr1: "my-attribute",
+      attr2: 123,
+    },
+  },
+  (span) => {
+    // do something that you want to measure
+    // once this is done, the span is automatically ended
+  }
+);
+```
+
+You can also pass an async function:
+
+```js
+Sentry.startSpan(
+  {
+    name: "my-span",
+    attributes: {},
+  },
+  async (span) => {
+    // do something that you want to measure
+    await waitOnSomething();
+    // once this is done, the span is automatically ended
+  }
+);
+```
+
+Since `startSpan()` will make the created span the active span, any automatic or manual instrumentation that creates
+spans inside of the callback will attach new spans as children of the span we just started.
+
+Note that if an error is thrown inside of the callback, the span status will automatically be set to be errored.
+
+### `startSpanManual()`
+
+This is a variation of `startSpan()` with the only change that it does not automatically end the span when the callback
+ends, but you have to call `span.end()` yourself:
```

Comment:
The start of the sentence can also be something else. I just thought it reads easier when having shorter sentences.

```suggestion
This is a variation of `startSpan()` with the only difference that it does not automatically end the span when the callback
ends. When using this, you have to call `span.end()` yourself:
```